### ISSUE TITLE
Imagenet pretrained weights

### ISSUE THREAD

**Issue by baoruxiao** [xiaochus/MobileNetV2#1]
Hi Xiaochus,

Would you mind sharing your imagenet pretrained weights?

**Comment by JonathanCMitchell** [comment]
Hi yes would you mind sharing them? I could really use them right now. Thanks!

**Comment by GustavZ** [comment]
Same here! Would be very useful!

**Comment by RadEdje** [comment]
Hi, I was hoping to retrain this mobileNetv2 with imagenet weights. So based on this thread it hasn't been trained on that correct? So i have to go the the main mobileNetV2.py from keras team at this github repo:

https://github.com/keras-team/keras/blob/master/keras/applications/mobilenetv2.py

and use that for re-training? thanks.

**Comment by alyato** [comment]
@xiaochus 
i m curious about these lines.

> x = GlobalAveragePooling2D()(x)
    x = Reshape((1, 1, 1280))(x)
    x = Dropout(0.3, name='Dropout')(x)
    x = Conv2D(k, (1, 1), padding='same')(x)
    x = Activation('softmax', name='softmax')(x)
    output = Reshape((k,))(x)

why it is this?
not

> x = GlobalAveragePooling2D()(x)
   x  =  Dense(k)(x)
    x = Activation('softmax', name='softmax')(x)
Do some guys give one explain?
thanks.
